modules/appeals_api/spec/requests/notice_of_disagreements/v0/notice_of_disagreements/evidence_submissions_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
```

Comment:
Added specs here are intended to replace the ones I removed from the decision reviews v2 spec files